projects/core/styles/taiga-ui-fonts.less

```diff
@@ -1 +1,3 @@
-@import (css) url('https://fonts.googleapis.com/css2?family=Manrope:wght@500;800&display=swap');
+// stylelint-disable function-url-quotes
```

Comment:
Maybe it is better to disable this stylint-rule ?

`url(...)` is used only in 2 `.less`-files of our project (50% does not satisfy this rule :peka:)

Reply:
nitpicking.

it is just up to you. You can keep it in this way.

Reply:
fixed